src/components/MonitorLog/ReviewSubmitTabPopup.jsx

```diff
@@ -0,0 +1,91 @@
+import {
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  useDisclosure,
+  Button,
+} from '@chakra-ui/react';
+import PropTypes from 'prop-types';
+import { FiCheck } from 'react-icons/fi';
+import { React, useState, useEffect } from 'react';
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'useEffect' is defined but never used.